src/components/JsonSchemaValidator/Schemas/dbtAttribution_0.1.0.js

```diff
@@ -0,0 +1,228 @@
+      group: 'Contexts, Filters, and Logs',
+      items: { type: 'string' },
+    },
+    snowplow__campaigns_to_include: {
+      type: 'array',
+      description: '> Click the plus sign to add a new entry',
+      minItems: 0,
+      title: 'Campaigns to Include',
+      longDescription: 'List of channels to include in the analysis (empty to keep all campaigns).',
+      packageDefault: '[ ] (no filter applied)',
+      group: 'Contexts, Filters, and Logs',
+      items: { type: 'string' },
+    },
+    snowplow__conversion_hosts: {
+      type: 'array',
+      description: '> Click the plus sign to add a new entry',
+      minItems: 0,
+      title: 'URL Hosts',
+      longDescription: '`url_hosts` to filter to in the data processing',
+      packageDefault: '[] (no filter applied)',
+      group: 'Contexts, Filters, and Logs',
+      items: { type: 'string' },
+    },
+    snowplow__consider_intrasession_channels: {
+      type: 'boolean',
+      group: 'Contexts, Filters, and Logs',
+      longDescription: 'If `false`, only considers the channel at the start of the session (i.e. first page view). If `true`, considers multiple channels in the conversion session as well as historically.',
+      packageDefault: 'false',
+      title: 'Consider Intrasession Channels',
+    },
+    snowplow__spend_source: {
+      title: 'Conversions Source',
+      description: 'Source of marketing spend table',
+      longDescription: 'The source (schema and table) of your marketing spend source. Optional, needed for the ROAS calculation of the snowplow_attribution_overview. Should be changed to a table reference with `spend` by `channel` and/or `campaign` by `spend_timestamp` (which denotes a timestamp field) information.',
+      type: 'string',
+      packageDefault: "{{ source('atomic', 'events') }}",
+      group: 'Warehouse and Tracker',
+    },
+    snowplow__conversion_stitching: {
+      type: 'boolean',
+      group: 'Contexts, Filters, and Logs',
+      longDescription: 'This should align with the variable of the same name inside the unfied package. If allowed it will consider the stitched_user_id field, not the user_identifier in the source data for more accurate results.',
+      packageDefault: 'false',
+      title: 'Conversion Stitching',
+    },
+    snowplow__conversion_clause: {
+      type: 'string',
+      title: 'Conversions Clause',
+      group: 'Operation and Logic',
+      longDescription: 'A string of sql to filter on certain conversion events.',
+      packageDefault: 'cv_value > 0',
+    },
+    snowplow__attribution_list: {
+      type: 'array',
+      description: '> Click the plus sign to add a new entry',
+      minItems: 1,
+      title: 'Attribution List',
+      longDescription: 'List of attribution types to use for reporting. Can be at least one of: first_touch, last_touch, linear, position_based).',
+      packageDefault: '[`first_touch`, `last_touch`, `linear`, `position_based`]',
```

Comment:
Mysterious closing bracket

Reply:
it was the ' vs ` again I think

Reply:
I meant in the description it ends with a closed bracket but you've never opened one